### ISSUE TITLE
Adjustable font size on the keyboard

### ISSUE THREAD

**Issue by jbell4588** [futo-org/android-keyboard#90]
While having understanding that the keyboard font-size was likely chosen to be as large as it is for accessibility reasons, it does seem like that might be better served through a user controlled setting, akin to the settings for keyboard height and keyboard offset.

**Comment by danielb2** [comment]
> Yes, please. The current default is too big.

Yes, it's too big because it takes up too much screen real estate.

**Comment by mufeedcm** [comment]
yeah +1 on this ,
and the size of the space bar and the other icons on that row are also pretty big, 
here's how futo looks,

![Image](https://github.com/user-attachments/assets/9d9a86e5-2ac6-49b0-b908-27dabe16d71b)

vs, this is how gboard looks,

![Image](https://github.com/user-attachments/assets/8acbd60a-01dc-4a31-97e9-23102b55fad4)

I think this will help reduce the use of more screen real estate :)

**Comment by mrfoxesite4482** [comment]
It's been two years and I still want this. Why is this not resolved yet? After two years?

**Comment by asparanoiks** [comment]
I've just installed FUTO and wanted to make the font size smaller, and surprise! No way to adjust it. And even more shocked now seeing that after TWO (edit: almost three lol) years after raising this, it's still not done... Just WOW.

Bumping this up, hoping I won't be waiting for another 2 years lol.

**Comment by danielb2** [comment]
given that I donated to this app, I think it would be cool if this could actually get addressed

**Comment by asparanoiks** [comment]
Tempted to do a PR for this myself, cause **ekhm** fuck Google and GBoard. Realistically I don't have the time right now, so I'm not promising anything, but if this is still open in a while and nobody's on it, I can take a look. Would much rather the FUTO team beat me to it though lol
